### PR TITLE
Org IdPs

### PR DESCRIPTION
**Pull request by pnzrr**
- Separates items into components 
- Full stub for IdPs with need to plug in data

image

### CODE REVIEW

**Review by xgp — CHANGES_REQUESTED** [review]
need to update the `updateIdentityProvider` api you're using. otherwise LGTM!

File: apps/admin-ui/src/orgs/useOrgFetcher.ts [hunk 1]
```diff
@@ -376,6 +382,48 @@ export default function useOrgFetcher(realm: string) {
     };
   }
 
+  // GET /:realm/orgs/:orgId/idps
+  async function getIdentityProviders(orgId: string) {
```
[the remark this answers is not in the document]
Comment (pnzrr) [reply apps/admin-ui/src/orgs/useOrgFetcher.ts:386]: Yeah this one isn't in use. It's `adminClient`. I'll pull this method.

File: apps/admin-ui/src/orgs/useOrgFetcher.ts [hunk 2]
```diff
@@ -376,6 +382,48 @@ export default function useOrgFetcher(realm: string) {
...
+
+    resp = await resp.json();
+    return {
+      error: true,
+      message: resp.error,
+    };
+  }
+
+  // PUT /:realm/orgs/:orgId/idps/:alias
+  async function updateIdentityProvider(
```
[the remark this answers is not in the document]
Comment (pnzrr) [reply apps/admin-ui/src/orgs/useOrgFetcher.ts:386]: Updated the API method to use the correct one.

**Comment by pnzrr** [comment]
Right now, there is a CORs error when trying to update from localhost. I was not able to fully verify yet.

**Review by xgp — APPROVED** [review]
:+1:

**Pull request merged**